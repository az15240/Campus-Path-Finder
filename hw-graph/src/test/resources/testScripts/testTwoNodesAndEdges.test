# Test a graph with two nodes and edges connecting them.
# Graph ('A') have one edge, graph ('B') have three edges pointing from the same node to the
# same targeted node, graph ('C') have three edges not necessarily pointing from the same node
# to the same targeted node.

CreateGraph A
AddNode A n1
AddNode A n2
ListChildren A n1
ListChildren A n2
AddEdge A n1 n2 e1
ListNodes A
ListChildren A n1
ListChildren A n2

CreateGraph B
AddNode B n1
AddNode B n2
ListNodes B
AddEdge B n1 n2 e1
ListChildren B n1
ListChildren B n2
AddEdge B n1 n2 e2
ListChildren B n1
ListChildren B n2
AddEdge B n1 n2 e3
ListChildren B n1
ListChildren B n2

CreateGraph C
AddNode C n1
AddNode C n2
ListNodes C
AddEdge C n1 n2 e1
ListChildren C n1
ListChildren C n2
AddEdge C n2 n1 e2
ListChildren C n1
ListChildren C n2
AddEdge C n1 n2 e3
ListChildren C n1
ListChildren C n2