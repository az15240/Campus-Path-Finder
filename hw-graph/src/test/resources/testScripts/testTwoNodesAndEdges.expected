# Test a graph with two nodes and edges connecting them.
# Graph ('A') have one edge, graph ('B') have three edges pointing from the same node to the
# same targeted node, graph ('C') have three edges not necessarily pointing from the same node
# to the same targeted node.

created graph A
added node n1 to A
added node n2 to A
the children of n1 in A are:
the children of n2 in A are:
added edge e1 from n1 to n2 in A
A contains: n1 n2
the children of n1 in A are: n2(e1)
the children of n2 in A are:

created graph B
added node n1 to B
added node n2 to B
B contains: n1 n2
added edge e1 from n1 to n2 in B
the children of n1 in B are: n2(e1)
the children of n2 in B are:
added edge e2 from n1 to n2 in B
the children of n1 in B are: n2(e1) n2(e2)
the children of n2 in B are:
added edge e3 from n1 to n2 in B
the children of n1 in B are: n2(e1) n2(e2) n2(e3)
the children of n2 in B are:

created graph C
added node n1 to C
added node n2 to C
C contains: n1 n2
added edge e1 from n1 to n2 in C
the children of n1 in C are: n2(e1)
the children of n2 in C are:
added edge e2 from n2 to n1 in C
the children of n1 in C are: n2(e1)
the children of n2 in C are: n1(e2)
added edge e3 from n1 to n2 in C
the children of n1 in C are: n2(e1) n2(e3)
the children of n2 in C are: n1(e2)